port/common/omrport.c

```diff
@@ -840,3 +841,4 @@ omrport_allocate_library(struct OMRPortLibrary **portLibrary)
 	}
 	return rc;
 }
+
```

Comment:
Please remove this extra blank line (the last two characters should be `"}\n"`).